lib/inc/internal/simple_config_origin.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <hocon/config_origin.hpp>
+#include <hocon/config_value.hpp>
```

Comment:
Why is this header file needed? It wasn't before, and nothing you added should require it.